### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by DmytrV**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -14,4 +14,9 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+    counter = 0
+    for character in phrase:
+        if character.lower() == letter.lower():
+            counter += 1
+
+    return counter
```
Comment (vsmutok) [inline app/main.py:22]: try to write in one line

Comment (vsmutok) [reply app/main.py:22]: Also you close your pull request 
Please reopen it

Comment (DmytrV) [reply app/main.py]: Pull request was reopened. 
The solution was written in one line.

**Review by vsmutok — APPROVED** [review]
Good Job!

**Pull request closed without merging**